src/app/onboarding/routes.ts

```diff
@@ -4,81 +4,54 @@ export const routes = (): RouteRecordRaw[] => {
     {
       path: '/onboarding',
       redirect: {
-        name: 'onboarding-welcome',
+        name: 'onboarding-welcome-view',
       },
-      component: () => import('@/app/onboarding/views/OnboardingView.vue'),
+      component: () => import('@/app/onboarding/views/OnboardingIndexView.vue'),
```

Comment:
I just saw the `Index` usage elsewhere also, so I guess this is intentional, so I'm fine either way here.

Reply:
There are two *IndexView files currently: MeshIndexView and OnboardingIndexView. They both have the kind of wrapping view approach as we do with the TabsView files. I’m not really sure what to call those. There are probably several options. Any suggestions?

I think it’s different from the control-planes and diagnostics module which both have a detail view at their root. The files in question here aren’t detail views.

Reply:
Honestly I really don't mind, if we aren't following the URL then I don't mind what these files are called, I'll still need to find the `routes.ts` and then follow the URL to import alias and then follow the import path. Ideally I wanted to remove all these `routes.ts` files and just follow the URL.

That being said, if we are to 'agree and commit' and the reasoning behind this was because several files containing the word `index` in them was problematic for searching, then we shouldn't be renaming files in the follow up PR with the word `index` in them, hence the query. Is this going to be problematic for you? Will we need to change this again in the short term? Considering its done elsewhere also it doesn't seem like a mistake, so if its intentional then I'm fine also.

So please go ahead and choose a name, then feel free to resolve.

Reply:
I’ve changed these two files to use *RootView instead of *IndexView.